src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/BlazorIdentityHelper.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.DotNet.Tools.Scaffold.AspNet.Helpers;
 
 internal static class BlazorIdentityHelper
```

Comment:
Really feels like BlazorIdentityHelper and IdentityHelper should share a lot of code. They're so similar.

Reply:
only one method in common but this could be one file with both identity helper methods. will add a TODO

Reply:
It looked like maybe GetTextTemplatingProperties (same name in both), GetApplicationUserTextTemplatingProperty (same name in both), BlazorIdentityTemplateTypes/IdentityTemplateTypes could be shared implementation with just different inputs (for instance if this wasn't a static class, but was perhaps a base class with two derived classes and the derived classes provided the values used by the base class implementations of those methods). But I haven't gone line by line to see if they really are close enough to make that worthwhile - was just at a glance.